src/development/tools/devtools/release-notes/release-notes-2.17.0-src.md

```diff
@@ -2,11 +2,37 @@
 
 Dart & Flutter DevTools - A Suite of Performance Tools for Dart and Flutter
 
-## General Updates
-This release included several bug fixes and improvements, as well work
-towards some new features that are coming soon!
-See the full list of commits for DevTools 2.17.0
-[in the changelog](https://github.com/flutter/devtools/blob/master/CHANGELOG.md#2170).
+## Inspector Updates
+* Added support for manually setting the package directories for your app. If you've ever 
+  loaded the Inspector and noticed that some of your widgets are not present in the 
+  Widget tree, this may be an indication that the package directories for your app 
+  have not been set or detected properly. Your package directories determine which 
+  widgets the Inspector considers to be from "your" application. If you see an 
+  empty Inspector widget tree, or if you develop widgets across multiple packages 
+  and want widgets from all these locations to show up in your tree, check the 
+  Inspector Settings dialog to ensure that your package directories are properly
+  configured. 
+  
+  [#4306](https://github.com/flutter/devtools/pull/4306)
+  
+<img
+  alt="package_directories"
+  src="{{site.url}}/development/tools/devtools/release-notes/images-2.17.0/package_directories.png"
+  width="800"
+/>
```

Comment:
can you use the image format that is used in the other release notes?

e.g.
```
![auto-complete-suggestions]({{site.url}}/development/tools/devtools/release-notes/images-2.14.0/image3.png "auto complete suggestions")
```

Reply:
the screen shot is much wider than page

the only way to narrow it in code os to use plain html
or, I can reduce resolution of the image (with little lost of quality)

Reply:
@johnpryan @sfshaza2 is there a best practice for image sizing?

Reply:
To be honest, the best approach is to optimize the image to make it smaller. However, we do prefer that you use the markdown method for resizing images. You can do that as follows:

```
![auto-complete-suggestions]({{site.url}}/development/tools/devtools/release-notes/images-2.14.0/image3.png "auto complete suggestions"){:width="90%"}
```